apps/condo/domains/organization/utils/accessSchema.js

```diff
@@ -1,191 +1,297 @@
-const isArray = require('lodash/isArray')
-const isEmpty = require('lodash/isEmpty')
+const get = require('lodash/get')
+const pick = require('lodash/pick')
 const uniq = require('lodash/uniq')
-const { validate: isUUID } = require('uuid')
 
-const { getByCondition, find } = require('@open-condo/keystone/schema')
+const conf = require('@open-condo/config')
+const { getRedisClient } = require('@open-condo/keystone/redis')
+const { find } = require('@open-condo/keystone/schema')
 
+const _redisClient = getRedisClient('default', 'cache')
+// NOTE: larger = better, but it can affect "after migration" state, where roles are changed via SQL
+const CACHE_TTL_IN_MS = 60 * 60 * 1000  // 1 hour in ms
+const DISABLE_USER_ORGANIZATION_CACHING = get(conf, 'DISABLE_USER_ORGANIZATION_CACHING', 'false').toLowerCase() === 'true'
```

Comment:
IMO configurable ttl could be kinda useful feature
Maybe join these two lines into single config?

Reply:
Added separate variable for it, JSON configs is a bit messy imho...